requirements-dev.txt

```diff
@@ -12,9 +12,9 @@ ruff==0.11.7
 testfixtures==8.3.0
 
 # Build and publish requirements
-cibuildwheel==2.23.0
 twine==6.1.0
-wheel==0.45.1
+build
```

Comment:
this is what was missing and causing the previous failure.

Reply:
@bmos, it think we should not pin here because build is an official pypa project and so i think we can safely just use the newest version.